src/app/state/importaccounts/index.ts

```diff
@@ -3,41 +3,65 @@ import { ErrorPayload } from 'types/errors'
 import { createSlice } from 'utils/@reduxjs/toolkit'
 import { ImportAccountsListAccount, ImportAccountsState, ImportAccountsStep } from './types'
 
-export const initialState: ImportAccountsState = { accounts: [], showAccountsSelectionModal: false }
+export const initialState: ImportAccountsState = {
+  accounts: [],
+  showAccountsSelectionModal: false,
+  accountsSelectionPageNumber: 0,
+  step: ImportAccountsStep.Idle,
+}
 
 const slice = createSlice({
   name: 'importAccounts',
   initialState,
   reducers: {
-    clear(state, action: PayloadAction<void>) {
+    clear(state, _action: PayloadAction<void>) {
```

Comment:
You can remove the _action parameters :D

Reply:
I have removed most of them, except this one: 
```
enumerateAccountsFromMnemonic(state, _action: PayloadAction<string>) 
```
... since keeping this is necessary to properly define the type of this action. (The string payload is used by the saga.)